Formula/joplin.rb

```diff
@@ -15,6 +16,7 @@ class Joplin < Formula
   depends_on "node"
 
   def install
+    inreplace "package.json", "\"sharp\": \"^0.22.1\",", "\"sharp\": \"^0.23.1\","
```

Comment:
At the very least this needs a comment describing why this patch is needed and when to remove it. Ideally upstream would fix this, but if that's not likely to happen soon we can keep this patch.